packages/auth/src/model/public_types.ts

```diff
@@ -202,6 +202,30 @@ export interface Auth {
    * @param persistence - The {@link Persistence} to use.
    */
   setPersistence(persistence: Persistence): Promise<void>;
+  /**
+   * Initializes the reCAPTCHA configuration on the `Auth` instance.
+   *
+   * @remarks
+   * This will initialize reCAPTCHA config of the current Auth session which affects the future auth
+   * requests.
+   *
+   * The reCAPTCHA config indicates whether the reCAPTCHA verification flow should be triggered for
+   * a specific auth provider. Note that this only affect the client auth request but won't override
```

Comment:
Doesn't initializeRecaptchaConfig() load the config from the server? How can it contradict the enrollment state?

Reply:
Hmm, good catch. This is no longer needed.